lib/rack/proxy.rb

```diff
@@ -42,8 +42,9 @@ def reconstruct_header_name(name)
     def initialize(opts = {})
       @streaming = opts.fetch(:streaming, true)
       @ssl_verify_none = opts.fetch(:ssl_verify_none, false)
-      @backend = URI(opts[:backend]) if opts[:backend]
+      @backend = opts[:backend] ? URI(opts[:backend]) : nil
       @read_timeout = opts.fetch(:read_timeout, 60)
+      @ssl_version = opts[:ssl_version] ? opts[:ssl_version] : nil
```

Comment:
Same here. Is it any different than just `@ssl_version = opts[:ssl_version]`? (Except for coercing 'falsy' values to nil)